src/olympia/shelves/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+from olympia.amo.models import ModelBase
+
+CHOICE_TYPES = (
+    ('category', 'category'),
+    ('collection', 'collection'),
+    ('extension', 'extension'),
+    ('recommended', 'recommended'),
+    ('search', 'search'),
+    ('theme', 'theme'),
+)
+
+
+class Shelf(ModelBase):
+    title = models.CharField(max_length=200, unique=True)
```

Comment:
don't set unique here - there's a possible scenario where Scott might want to have two shelves with the same title temporarily (maybe creating a replacement while keeping the old shelf live).